cluster/genesis.go

```diff
@@ -836,10 +21,32 @@ func (client *cluster) generateGenesis() error {
 	if err != nil {
 		return err
 	}
-	var b bytes.Buffer
-	genesisTpl.Execute(&b, map[string]string{
-		"NetworkID": client.NetworkID,
-	})
+
+	newGenesis, err := genesis.GenerateGenesis(
+		genesis.GenesisOptions{
+			Network:                       "test",
+			MaxNumValidators:              "1",
+			UnbondingPeriod:               "0",
+			WalletAddressGenesisValidator: "0xd6e579085c82329c89fca7a9f012be59028ed53f",
+			PrefundedAccounts: []genesis.PrefundedAccount{
+				{
+					WalletAddress: "0xd6e579085c82329c89fca7a9f012be59028ed53f",
+					Balance:       "0x200000000000000000000000000000000000000000000000000000000000000",
+				},
+				{
+					WalletAddress: "0x259be75d96876f2ada3d202722523e9cd4dd917d",
+					Balance:       "1000000000000000000",
+				},
+			},
+			SmartContractsOwner: "0x259be75d96876f2ada3d202722523e9cd4dd917d",
```

Comment:
This is probably going to be multisig: meaning that a contract will probably have multiple keys as the owner - please open an issue for this specific problem. We need to check this! :) For now we can leave it as is.

Reply:
SmartContractsOwner is too broad. I'd call it CoreContractsOwner